2 - RTB/Lettera di presentazione/Lettera di presentazione RTB.typ

```diff
@@ -0,0 +1,104 @@
+#set text(
+  font: "New Computer Modern",
+  lang: "it",
+)
+#set par(
+  leading: 0.85em,
+)
+#set list(
+  marker: ([•], [--]),
+)
+#set table(
+  fill: (_, row) => if row == 0 { luma(220) },
+  stroke: 0.5pt + luma(140)
+)
+#show link: set text(fill: blue)
+#show link: it => [
+  #it
+  #v(0.75em)
+]
+#show "WIP": it => [
+  #text(it, fill: red)
+]
+
+#v(-1em)
+#align(
+  end,
+  block(
+    width: 50%,
+    grid(
+      columns: (3fr, 2fr),
+      column-gutter: 1em,
+      [
+        #align(
+          end + horizon,
+          [
+            #text(
+              size: 1.2em,
+              weight: "bold",
+              "Error_418"
+            )
+            #v(-0.5em)
+            #link("mailto:error418swe@gmail.com")
+          ]
+        )
+      ],
+      [
+        #align(
+          horizon,
+          [
+            #v(-1em)
+            #image("/logo.png", width: 100%)
+          ]
+        )
+      ]
+    )
+  )
+)
+#v(2em)
+
+Egregio Prof. Tullio Vardanega,\
+Egregio Prof. Riccardo Cardin,
+
+#v(1em)
+
+Con la presente, il gruppo Error-418 intende comunicarVi la volontà di candidarsi alla revisione per la _Requirements and Tecnology Baseline_. Sarà esposto lo stato di avanzamento del progetto:
+
+#align(center)[*"WMS3: Warehouse Management 3D"*]
+
+proposto dall'azienda _Sanmarco Informatica S.p.a._, altresì denominato Capitolato 5.
+
```

Comment:
Inserirei il link al capitolato, Aggiudacato C5: https://www.math.unipd.it/~tullio/IS-1/2023/Progetto/C5.pdf

Reply:
Ci starebbe ma inserendo il link lui mi va a capo e si rompe il layout. Non ho tempo di fixare